src/remote/remote.go

```diff
@@ -272,7 +273,7 @@ func (c *Client) initFetch() error {
 	if c.state.Config.Remote.Secure {
 		dialOpts = append(dialOpts, grpc.WithTransportCredentials(credentials.NewClientTLSFromCert(nil, "")))
 	} else {
-		dialOpts = append(dialOpts, grpc.WithInsecure())
+		dialOpts = append(dialOpts, grpc.WithTransportCredentials(insecure.NewCredentials()))
```

Comment:
A quite migration for a deprecated function

Reply:
Why do we want to do this [without transport security?](https://pkg.go.dev/google.golang.org/grpc/credentials/insecure#NewCredentials)

Reply:
There are some use cases where it appears unencrypted to the application - maybe the most legit is going via a service mesh (e.g. Istio) where the connection will be encrypted for us.
The default is that TLS is on though so a user has to explicitly opt out.